blocks/library/shortcode/editor.scss

```diff
@@ -13,7 +13,7 @@
 	}
 
 	.blocks-plain-text {
-		flex: 1 0 0%;
+		flex: 1 0 0;
```

Comment:
This is a very obscure Flex bug affecting IE11: https://github.com/philipwalton/flexbugs#flexbug-4

Reply:
Coolio, I'll revert and add a comment

Reply:
Fixed in a83363d608f010da2caef1c45a1d9f76dbabf2af.